### ISSUE TITLE
Deliver step gets stuck while running Transporter

### ISSUE THREAD

**Issue by twomedia** [fastlane/fastlane#19428]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
Whenever we try and Deliver our iOS apps to iTunes Connect fastlane becomes stuck while the Transporter logic is running. As far as I can tell there are no errors outputted even in verbose mode. We left a build submission running for a few days and nothing changed. 

If we try and submit the exact same binary using the Transporter app directly the build is uploaded without any issues.

##### Command executed
`fastlane beta submit:false --verbose`

##### Complete output when running fastlane, including the stack trace and command used
Note: Any redacted information was replaced with `**********`
<details>
  <pre> [03:10:12] :	 [Step 6/11] INFO [2021-10-04 14:10:12.19]:  [32m--------------------- [0m
[03:10:12] :	 [Step 6/11] INFO [2021-10-04 14:10:12.19]:  [32m--- Step: deliver --- [0m
[03:10:12] :	 [Step 6/11] INFO [2021-10-04 14:10:12.19]:  [32m--------------------- [0m
[03:10:12] :	 [Step 6/11] INFO [2021-10-04 14:10:12.19]: Login to App Store Connect (**********)
[03:10:12] :	 [Step 6/11] Reading keychain entry, because either user or password were empty
[03:10:12] :	 [Step 6/11] Loading session from '/Users/**********/.fastlane/spaceship/**********/cookie'
[03:10:13] :	 [Step 6/11] Looking for App Store Connect Team with ID **********
[03:10:14] :	 [Step 6/11] INFO [2021-10-04 14:10:14.23]: Login successful
[03:10:22] :	 [Step 6/11] INFO [2021-10-04 14:10:22.99]:  [36m$ xar -C /Volumes/DataDrive/TeamCity/BuildAgent2/temp/buildTmp/d20211004-37762-96eoxc -xf **********.pkg [0m
[03:10:24] :	 [Step 6/11] 
[03:10:24] :	 [Step 6/11] +--------------------------------------+-----------------------------+
[03:10:24] :	 [Step 6/11] |                       [32mdeliver 2.195.0 Summary [0m                       |
[03:10:24] :	 [Step 6/11] +--------------------------------------+-----------------------------+
[03:10:24] :	 [Step 6/11] | submit_for_review                    | false                       |
[03:10:24] :	 [Step 6/11] | force                                | true                        |
[03:10:24] :	 [Step 6/11] | skip_screenshots                     | true                        |
[03:10:24] :	 [Step 6/11] | skip_metadata                        | true                        |
[03:10:24] :	 [Step 6/11] | platform                             | osx                         |
[03:10:24] :	 [Step 6/11] | screenshots_path                     | ./fastlane/screenshots      |
[03:10:24] :	 [Step 6/11] | metadata_path                        | ./fastlane/metadata         |
[03:10:24] :	 [Step 6/11] | app_version                          | 0.986.3                     |
[03:10:24] :	 [Step 6/11] | username                             | **********         |
[03:10:24] :	 [Step 6/11] | app_identifier                       | ********** |
[03:10:24] :	 [Step 6/11] | pkg                                  | **********.pkg         |
[03:10:24] :	 [Step 6/11] | edit_live                            | false                       |
[03:10:24] :	 [Step 6/11] | use_live_version                     | false                       |
[03:10:24] :	 [Step 6/11] | skip_binary_upload                   | false                       |
[03:10:24] :	 [Step 6/11] | skip_app_version_update              | false                       |
[03:10:24] :	 [Step 6/11] | overwrite_screenshots                | false                       |
[03:10:24] :	 [Step 6/11] | sync_screenshots                     | false                       |
[03:10:24] :	 [Step 6/11] | reject_if_possible                   | false                       |
[03:10:24] :	 [Step 6/11] | phased_release                       | false                       |
[03:10:24] :	 [Step 6/11] | reset_ratings                        | false                       |
[03:10:24] :	 [Step 6/11] | team_id                              | **********                      |
[03:10:24] :	 [Step 6/11] | dev_portal_team_id                   | **********                  |
[03:10:24] :	 [Step 6/11] | run_precheck_before_submit           | true                        |
[03:10:24] :	 [Step 6/11] | precheck_default_rule_level          | warn                        |
[03:10:24] :	 [Step 6/11] | ignore_language_directory_validation | false                       |
[03:10:24] :	 [Step 6/11] | precheck_include_in_app_purchases    | true                        |
[03:10:24] :	 [Step 6/11] +--------------------------------------+-----------------------------+
[03:10:24] :	 [Step 6/11] 
[03:10:24] :	 [Step 6/11] INFO [2021-10-04 14:10:24.02]: Making sure the latest version on App Store Connect matches '0.986.3'...
[03:10:24] :	 [Step 6/11] INFO [2021-10-04 14:10:24.94]:  [32m'0.986.3' is the latest version on App Store Connect [0m
[03:10:24] :	 [Step 6/11] INFO [2021-10-04 14:10:24.95]: Uploading binary to App Store Connect
[03:10:25] :	 [Step 6/11] INFO [2021-10-04 14:10:25.32]:  [32mWrote XML data to '/tmp/**********.itmsp' [0m
[03:10:25] :	 [Step 6/11] INFO [2021-10-04 14:10:25.32]: Fetching password for transporter from environment variable named `FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD`
[03:10:25] :	 [Step 6/11] INFO [2021-10-04 14:10:25.89]: Going to upload updated app to App Store Connect
[03:10:25] :	 [Step 6/11] INFO [2021-10-04 14:10:25.89]:  [32mThis might take a few minutes. Please don't interrupt the script. [0m
[03:10:25] :	 [Step 6/11] DEBUG [2021-10-04 14:10:25.89]: ITMS_TRANSPORTER_PASSWORD=YourPassword xcrun iTMSTransporter -m upload -u ********** -p @env:ITMS_TRANSPORTER_PASSWORD -f /tmp/**********.itmsp -k 100000 2>&1
[03:10:30] :	 [Step 6/11] DEBUG [2021-10-04 14:10:30.09]: [Transporter]: INFO: Configuring logging...
[03:10:30] :	 [Step 6/11] DEBUG [2021-10-04 14:10:30.10]: [Transporter]: INFO: Logging level set to eXtreme
[03:10:31] :	 [Step 6/11] DEBUG [2021-10-04 14:10:31.87]: [Transporter]: DEBUG: SMART-CLIENT: initializing data center locking ops: [authenticateForSession, validateAssets]
[03:10:32] :	 [Step 6/11] DEBUG [2021-10-04 14:10:32.21]: [Transporter]: DEBUG: SMART-CLIENT: initializing data centers with: [contentdelivery01.itunes.apple.com]
[03:10:32] :	 [Step 6/11] DEBUG [2021-10-04 14:10:32.21]: [Transporter]: INFO: iTMSTransporter : iTunes Store Transporter [2.2.0]
[03:10:32] :	 [Step 6/11] DEBUG [2021-10-04 14:10:32.21]: [Transporter]: INFO: OS identifier: Mac OS X 11.5.1 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-5; jre=14.0.2+12-iTunesOpenJDK-5
[03:10:32] :	 [Step 6/11] DEBUG [2021-10-04 14:10:32.21]: [Transporter]: INFO: Memory: [JVM] 999M free, 1024M total, 2048M max [System] (Physical) 100693M free, 163840M total (Swap) 0 free, 0 total
[03:10:32] :	 [Step 6/11] DEBUG [2021-10-04 14:10:32.22]: [Transporter]: DBG-X: Memory: [JVM] 999M free, 1024M total, 2048M max [System] (Physical) 100691M free, 163840M total (Swap) 0 free, 0 total
[03:10:32] :	 [Step 6/11] DEBUG [2021-10-04 14:10:32.22]: [Transporter]: DBG-X: Using operation named: authenticateForSession
[03:10:32] :	 [Step 6/11] DEBUG [2021-10-04 14:10:32.24]: [Transporter]: DBG-X: Apple's web service operation input parameters:
[03:10:32] :	 [Step 6/11] DEBUG [2021-10-04 14:10:32.24]: [Transporter]: DBG-X:   parameter Application = iTMSTransporter
[03:10:32] :	 [Step 6/11] DEBUG [2021-10-04 14:10:32.24]: [Transporter]: DBG-X:   parameter BaseVersion = 2.1.0
[03:10:32] :	 [Step 6/11] DEBUG [2021-10-04 14:10:32.24]: [Transporter]: DBG-X:   parameter OSIdentifier = Mac OS X 11.5.1 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-5; jre=14.0.2+12-iTunesOpenJDK-5
[03:10:32] :	 [Step 6/11] DEBUG [2021-10-04 14:10:32.24]: [Transporter]: DBG-X:   parameter StatisticsClientStartDateTimeZoneISO = 2021-10-04T14:10:32+11:00
[03:10:32] :	 [Step 6/11] DEBUG [2021-10-04 14:10:32.24]: [Transporter]: DBG-X:   parameter StatisticsPreviousCallDurationInSecs = 0.437020667
[03:10:32] :	 [Step 6/11] DEBUG [2021-10-04 14:10:32.24]: [Transporter]: DBG-X:   parameter TransporterArguments = -m upload -u ********** -p **hidden value** -f /tmp/**********.itmsp -k 100000
[03:10:32] :	 [Step 6/11] DEBUG [2021-10-04 14:10:32.24]: [Transporter]: DBG-X:   parameter Version = 2.2.0
[03:10:32] :	 [Step 6/11] DEBUG [2021-10-04 14:10:32.24]: [Transporter]: DBG-X:   parameter iTMSTransporterMode = upload
[03:10:32] :	 [Step 6/11] DEBUG [2021-10-04 14:10:32.24]: [Transporter]: INFO: id = 20211004141032-581
[03:10:32] :	 [Step 6/11] DEBUG [2021-10-04 14:10:32.24]: [Transporter]: DEBUG: SMART-CLIENT: getCurrent(); current data center is null
[03:10:32] :	 [Step 6/11] DEBUG [2021-10-04 14:10:32.24]: [Transporter]: DEBUG: SMART-CLIENT: getCurrent(); iterator was null, setting it
[03:10:32] :	 [Step 6/11] DEBUG [2021-10-04 14:10:32.24]: [Transporter]: DEBUG: SMART-CLIENT: getCurrent(); setting current data center to: contentdelivery01.itunes.apple.com
[03:10:32] :	 [Step 6/11] DEBUG [2021-10-04 14:10:32.25]: [Transporter]: INFO: iTMSTransporter Correlation Key: ad003644-aab4-4418-b889-6e586cb3ebae-0001
[03:10:32] :	 [Step 6/11] DEBUG [2021-10-04 14:10:32.25]: [Transporter]: DEBUG: SMART-CLIENT: Host HTTP header: contentdelivery01.itunes.apple.com
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.25]: [Transporter]: DEBUG: SMART-CLIENT: pinning data center to: contentdelivery01.itunes.apple.com
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.25]: [Transporter]: DBG-X: Apple's web service operation return value:
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.25]: [Transporter]: DBG-X:   parameter SharedSecret = **hidden value**
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.25]: [Transporter]: DBG-X:   parameter MultipartUploadsEnabled = true
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.25]: [Transporter]: DBG-X:   parameter EnableJWTForAllCalls = false
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.25]: [Transporter]: DBG-X:   parameter TxHeaders = {jenga=true}
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.25]: [Transporter]: DBG-X:   parameter ShouldUseFeatures = (null)
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.25]: [Transporter]: DBG-X:   parameter SessionExpiration = 2021-10-08T03:10:33.062Z
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.25]: [Transporter]: DBG-X:   parameter SessionId = **********
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.25]: [Transporter]: DBG-X:   parameter ShouldUseRESTAPIs = false
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.25]: [Transporter]: DBG-X:   parameter StreamingSettings = {LogStreamingEnabled=true, MultipartUploadsEnabled=true, AssetDescriptionStreamingEnabled=false}
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.26]: [Transporter]: INFO: Upload mode selected.
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.27]: [Transporter]: DBG-X: Memory: [JVM] 980M free, 1024M total, 2048M max [System] (Physical) 100678M free, 163840M total (Swap) 0 free, 0 total
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.27]: [Transporter]: DBG-X: Using operation named: authenticateForTransportDiscovery
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.27]: [Transporter]: DBG-X: Apple's web service operation input parameters:
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.27]: [Transporter]: DBG-X:   parameter Application = iTMSTransporter
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.27]: [Transporter]: DBG-X:   parameter BaseVersion = 2.1.0
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.27]: [Transporter]: DBG-X:   parameter OSIdentifier = Mac OS X 11.5.1 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-5; jre=14.0.2+12-iTunesOpenJDK-5
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.27]: [Transporter]: DBG-X:   parameter StatisticsClientStartDateTimeZoneISO = 2021-10-04T14:10:33+11:00
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.27]: [Transporter]: DBG-X:   parameter TransporterArguments = -m upload -u ********** -p **hidden value** -f /tmp/**********.itmsp -k 100000
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.27]: [Transporter]: DBG-X:   parameter Version = 2.2.0
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.27]: [Transporter]: DBG-X:   parameter iTMSTransporterMode = upload
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.27]: [Transporter]: INFO: id = 20211004141033-703
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.27]: [Transporter]: DEBUG: SMART-CLIENT: getCurrent(); using pinned data center = contentdelivery01.itunes.apple.com
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.27]: [Transporter]: INFO: iTMSTransporter Correlation Key: ad003644-aab4-4418-b889-6e586cb3ebae-0001
[03:10:33] :	 [Step 6/11] DEBUG [2021-10-04 14:10:33.28]: [Transporter]: DEBUG: SMART-CLIENT: Host HTTP header: contentdelivery01.itunes.apple.com
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.12]: [Transporter]: DBG-X: Apple's web service operation return value:
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.12]: [Transporter]: DBG-X:   parameter EnableJWTForAllCalls = false
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.12]: [Transporter]: DBG-X:   parameter PreferredTransports = [HTTP, Signiant, Aspera, DAV]
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.12]: [Transporter]: DBG-X:   parameter TransportHosts.Signiant = [sgr602.apple.com:44001, sgr601.apple.com:44001, sgr603.apple.com:44001, sgr604.apple.com:44001]
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.12]: [Transporter]: DBG-X:   parameter ShouldUseFeatures = (null)
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.12]: [Transporter]: DBG-X:   parameter SessionExpiration = 2021-10-08T03:10:33.876Z
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.12]: [Transporter]: DBG-X:   parameter TransportHosts.DAV = [itmsdav.apple.com:443]
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.12]: [Transporter]: DBG-X:   parameter TransportHosts.Aspera = [vgr502.apple.com:33001, vgr504.apple.com:33001, vgr503.apple.com:33001, vgr501.apple.com:33001]
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.12]: [Transporter]: DBG-X:   parameter ShouldUseRESTAPIs = false
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.12]: [Transporter]: DBG-X:   parameter StreamingSettings = {LogStreamingEnabled=true, MultipartUploadsEnabled=true, AssetDescriptionStreamingEnabled=false}
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.13]: [Transporter]: INFO: Transporter's command line arguments are: -m upload -u ********** -p **hidden value** -f /tmp/**********.itmsp -k 100000
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.13]: [Transporter]: INFO: Gathering the list of valid files from the package ...
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.13]: [Transporter]: DEBUG:   metadata.xml will be verified by Apple's web service to determine if it is a valid file.
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.13]: [Transporter]: DEBUG:   a2c1b31e42fff2ba55ef7614665972b9.pkg will be verified by Apple's web service to determine if it is a valid file.
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.13]: [Transporter]: INFO: Finished gathering the list of valid files from the package.
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.13]: [Transporter]: INFO: Examining the package at: /tmp/**********.itmsp
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.13]: [Transporter]: INFO: Ensuring that package has well formed metadata file...
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.13]: [Transporter]: DEBUG: Getting case-sensitive metadata xml filename...
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.13]: [Transporter]: DEBUG:   Case-sensitive metadata filename is: metadata.xml
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.14]: [Transporter]: INFO: Performing authentication of package **********.itmsp ...
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.15]: [Transporter]: DBG-X: Memory: [JVM] 965M free, 1024M total, 2048M max [System] (Physical) 100666M free, 163840M total (Swap) 0 free, 0 total
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.16]: [Transporter]: DBG-X: Deflated 461 bytes to 408
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.16]: [Transporter]: DBG-X: Using operation named: validateMetadata
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.16]: [Transporter]: DBG-X: Apple's web service operation input parameters:
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.16]: [Transporter]: DBG-X:   parameter Application = iTMSTransporter
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.16]: [Transporter]: DBG-X:   parameter BaseVersion = 2.1.0
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.16]: [Transporter]: DBG-X:   parameter Files = [metadata.xml, a2c1b31e42fff2ba55ef7614665972b9.pkg]
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.16]: [Transporter]: DBG-X:   parameter Metadata = <?xml version="1.0" encoding="UTF-8"?>
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.16]: [Transporter]: DBG-X:   parameter MetadataChecksum = **********
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.16]: [Transporter]: DBG-X:   parameter MetadataCompressed = (suppressed)
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.16]: [Transporter]: DBG-X:   parameter MetadataInfo = {app_platform=osx, primary_bundle_identifier=, device_id=, bundle_identifier=, packageVersion=software5.4, apple_id=321756558, asset_types=[product-archive], bundle_version=, bundle_short_version_string=}
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.16]: [Transporter]: DBG-X:   parameter OSIdentifier = Mac OS X 11.5.1 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-5; jre=14.0.2+12-iTunesOpenJDK-5
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.16]: [Transporter]: DBG-X:   parameter PackageName = **********.itmsp
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.16]: [Transporter]: DBG-X:   parameter PackageSize = 147795333
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.16]: [Transporter]: DBG-X:   parameter StatisticsClientStartDateTimeZoneISO = 2021-10-04T14:10:34+11:00
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.16]: [Transporter]: DBG-X:   parameter Throttle = 100000
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.16]: [Transporter]: DBG-X:   parameter TransporterArguments = -m upload -u ********** -p **hidden value** -f /tmp/**********.itmsp -k 100000
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.16]: [Transporter]: DBG-X:   parameter Version = 2.2.0
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.16]: [Transporter]: DBG-X:   parameter iTMSTransporterMode = upload
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.16]: [Transporter]: INFO: id = 20211004141034-300
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.16]: [Transporter]: DEBUG: SMART-CLIENT: getCurrent(); using pinned data center = contentdelivery01.itunes.apple.com
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.16]: [Transporter]: INFO: iTMSTransporter Correlation Key: ad003644-aab4-4418-b889-6e586cb3ebae-0001
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.16]: [Transporter]: DEBUG: SMART-CLIENT: Host HTTP header: contentdelivery01.itunes.apple.com
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.86]: [Transporter]: DBG-X: Apple's web service operation return value:
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.86]: [Transporter]: DBG-X:   parameter EnableJWTForAllCalls = false
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.86]: [Transporter]: DBG-X:   parameter ShouldUseFeatures = (null)
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.86]: [Transporter]: DBG-X:   parameter SessionExpiration = 2021-10-08T03:10:34.328Z
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.86]: [Transporter]: DBG-X:   parameter MinimumFileSizeThresholdForCheckum = 104857600
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.86]: [Transporter]: DBG-X:   parameter SoftwareMediaDescriptionFormat = binary
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.86]: [Transporter]: DBG-X:   parameter SoftwareMediaDescriptionThresholds = {in-app-purchase-content={extensions=[pkg], size=100000000}, ccats={extensions=[pdf], size=100000000}, product-archive={extensions=[pkg], size=100000000}, developer-id-package={extensions=[zip, pkg, dmg], size=100000000}, bundle={extensions=[zip, ipa], size=100000000}, large-icon={extensions=[jpg, png, jpeg], size=100000000}}
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.86]: [Transporter]: DBG-X:   parameter Success = true
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.86]: [Transporter]: DBG-X:   parameter StreamingSettings = {LogStreamingEnabled=true, MultipartUploadsEnabled=true, AssetDescriptionStreamingEnabled=false}
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.86]: [Transporter]: DBG-X:   parameter FilesToUpload = [metadata.xml, a2c1b31e42fff2ba55ef7614665972b9.pkg]
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.86]: [Transporter]: DBG-X:   parameter AssetsToDescribe = [{Role=source, Filename=a2c1b31e42fff2ba55ef7614665972b9.pkg, ShouldDescribeAsset=true, UTI=com.apple.pkg}]
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.86]: [Transporter]: DBG-X:   parameter CPUToken = **********/**********
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.86]: [Transporter]: DBG-X:   parameter Checksums = {a2c1b31e42fff2ba55ef7614665972b9.pkg=**********}
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.86]: [Transporter]: DBG-X:   parameter FileSizes = {a2c1b31e42fff2ba55ef7614665972b9.pkg=147794872}
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.86]: [Transporter]: DBG-X:   parameter ShouldUseRESTAPIs = false
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.86]: [Transporter]: DBG-X:   parameter ProviderNameForToken = **********
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.87]: [Transporter]: INFO: The list of files requested for upload by Apple is: [metadata.xml, a2c1b31e42fff2ba55ef7614665972b9.pkg].  These are the files that will be uploaded.
[03:10:34] :	 [Step 6/11] DEBUG [2021-10-04 14:10:34.87]: [Transporter]: INFO: Starting media analysis of assets
[03:10:35] :	 [Step 6/11] DEBUG [2021-10-04 14:10:35.04]: [Transporter]: DEBUG: Handling asset: filename=a2c1b31e42fff2ba55ef7614665972b9.pkg,uti=com.apple.pkg,role=source,describe=true
[03:10:35] :	 [Step 6/11] DEBUG [2021-10-04 14:10:35.04]: [Transporter]: DEBUG: Using container type tool name and version: Mac App Describer:1.26
[03:10:35] :	 [Step 6/11] DEBUG [2021-10-04 14:10:35.04]: [Transporter]: DEBUG: Using path: /tmp/**********.itmsp/a2c1b31e42fff2ba55ef7614665972b9.pkg
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.30]: [Transporter]: DBG-X: description length:0
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.30]: [Transporter]: DBG-X: Asset description: 
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.30]: [Transporter]: DEBUG: Asset successfully described: com.apple.jingle.leghorn.software.AbstractSoftwareDescriber$1@28b523a
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.38]: [Transporter]: DBG-X: Deflated 614 bytes to 389
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.38]: [Transporter]: INFO: Asset media analysis has completed
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.38]: [Transporter]: DBG-X: Memory: [JVM] 934M free, 1024M total, 2048M max [System] (Physical) 100635M free, 163840M total (Swap) 0 free, 0 total
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.38]: [Transporter]: DBG-X: Using operation named: lookupTransportDiagnostic
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.38]: [Transporter]: DBG-X: Apple's web service operation input parameters:
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.38]: [Transporter]: DBG-X:   parameter Application = iTMSTransporter
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.38]: [Transporter]: DBG-X:   parameter BaseVersion = 2.1.0
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.38]: [Transporter]: DBG-X:   parameter OSIdentifier = Mac OS X 11.5.1 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-5; jre=14.0.2+12-iTunesOpenJDK-5
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.38]: [Transporter]: DBG-X:   parameter StatisticsClientStartDateTimeZoneISO = 2021-10-04T14:10:47+11:00
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.38]: [Transporter]: DBG-X:   parameter TransporterArguments = -m upload -u ********** -p **hidden value** -f /tmp/**********.itmsp -k 100000
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.38]: [Transporter]: DBG-X:   parameter TxTransportVersions = {SigniantTCP=1.2.0, Aspera=1.2.1, DAV=1.2.0, Signiant=1.2.0, HTTP=1.4.3}
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.38]: [Transporter]: DBG-X:   parameter Version = 2.2.0
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.38]: [Transporter]: DBG-X:   parameter iTMSTransporterMode = upload
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.38]: [Transporter]: INFO: id = 20211004141047-863
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.38]: [Transporter]: DEBUG: SMART-CLIENT: getCurrent(); using pinned data center = contentdelivery01.itunes.apple.com
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.38]: [Transporter]: INFO: iTMSTransporter Correlation Key: ad003644-aab4-4418-b889-6e586cb3ebae-0001
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.38]: [Transporter]: DEBUG: SMART-CLIENT: Host HTTP header: contentdelivery01.itunes.apple.com
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.75]: [Transporter]: DBG-X: Apple's web service operation return value:
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.75]: [Transporter]: DBG-X:   parameter AsperaLoggingLevel = OFF
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.75]: [Transporter]: DBG-X:   parameter EnableJWTForAllCalls = false
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.75]: [Transporter]: DBG-X:   parameter SigniantLoggingLevel = OFF
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.75]: [Transporter]: DBG-X:   parameter ShouldUseFeatures = (null)
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.75]: [Transporter]: DBG-X:   parameter SessionExpiration = 2021-10-08T03:10:47.549Z
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.75]: [Transporter]: DBG-X:   parameter ShouldUseRESTAPIs = false
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.75]: [Transporter]: DBG-X:   parameter StreamingSettings = {LogStreamingEnabled=true, MultipartUploadsEnabled=true, AssetDescriptionStreamingEnabled=false}
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.77]: [Transporter]: DEBUG: SMART-CLIENT: getCurrent(); using pinned data center = contentdelivery01.itunes.apple.com
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.77]: [Transporter]: DBG-X: Using operation named: createTestReservation
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.77]: [Transporter]: DBG-X: Apple's web service operation input parameters:
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.78]: [Transporter]: DBG-X:   parameter Application = iTMSTransporter
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.78]: [Transporter]: DBG-X:   parameter BaseVersion = 2.1.0
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.78]: [Transporter]: DBG-X:   parameter fileDescriptions = ( one FileDescription )
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.78]: [Transporter]: DBG-X:   parameter iTMSTransporterMode = upload
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.78]: [Transporter]: DBG-X:   parameter NewPackageName = http-transport-diagnostic-20211004031047-775-8259.itmsp
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.78]: [Transporter]: DBG-X:   parameter OSIdentifier = Mac OS X 11.5.1 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-5; jre=14.0.2+12-iTunesOpenJDK-5
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.78]: [Transporter]: DBG-X:   parameter TransporterArguments = -m upload -u ********** -p **hidden value** -f /tmp/**********.itmsp -k 100000
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.78]: [Transporter]: DBG-X:   parameter Username = **********
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.78]: [Transporter]: DBG-X:   parameter Version = 2.2.0
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.78]: [Transporter]: INFO: id = 20211004141047-735
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.87]: [Transporter]: DEBUG: SMART-CLIENT: getCurrent(); using pinned data center = contentdelivery01.itunes.apple.com
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.87]: [Transporter]: INFO: iTMSTransporter Correlation Key: ad003644-aab4-4418-b889-6e586cb3ebae-0001
[03:10:47] :	 [Step 6/11] DEBUG [2021-10-04 14:10:47.87]: [Transporter]: DEBUG: DataCenter: contentdelivery01.itunes.apple.com
[03:10:49] :	 [Step 6/11] DEBUG [2021-10-04 14:10:49.03]: [Transporter]: DBG-X: Apple's web service operation return value:
[03:10:49] :	 [Step 6/11] DEBUG [2021-10-04 14:10:49.03]: [Transporter]: DBG-X:   parameter Reservations = ( one Reservation )
[03:10:49] :	 [Step 6/11] DEBUG [2021-10-04 14:10:49.03]: [Transporter]: DBG-X:   parameter SessionExpiration = 2021-10-08T03:10:48.463Z
[03:10:49] :	 [Step 6/11] DEBUG [2021-10-04 14:10:49.03]: [Transporter]: DBG-X:   parameter Success = true
[03:10:49] :	 [Step 6/11] DEBUG [2021-10-04 14:10:49.04]: [Transporter]: INFO: Setting transport log file: /var/folders/y8/wsp5cvc16pz39qkpy5wb0y9w0000gn/T/http-transport-diagnostic-20211004031047-775-8259.itmsp13919854988931249073.log
[03:10:49] :	 [Step 6/11] DEBUG [2021-10-04 14:10:49.04]: [Transporter]: INFO: Starting upload for packageName: http-transport-diagnostic-20211004031047-775-8259.itmsp
[03:10:49] :	 [Step 6/11] DEBUG [2021-10-04 14:10:49.04]: [Transporter]: INFO: Thread Pool Configuration: {rateLimit=100000 KBs, minConnections=6, coreThreads=6, perCoreThreadMultiplier=1.65, processors=24, maxThreads=14, maxConnections=13}
[03:10:49] :	 [Step 6/11] DEBUG [2021-10-04 14:10:49.04]: [Transporter]: INFO: Reservation CPUSource/v4/25/63/1a/25631acf-376e-7398-6a8c-e1840347743e/http-transport-diagnostic-2808097841743991954.txt created for file: (http-transport-diagnostic-2808097841743991954.txt), part count:(1), file size: (199)
[03:10:49] :	 [Step 6/11] DEBUG [2021-10-04 14:10:49.05]: [Transporter]: INFO: Computing total size of files to upload ...
[03:10:49] :	 [Step 6/11] DEBUG [2021-10-04 14:10:49.05]: [Transporter]: DBG-X:   adding file for % complete logging: http-transport-diagnostic-2808097841743991954.txt , size: 199
[03:10:49] :	 [Step 6/11] DEBUG [2021-10-04 14:10:49.05]: [Transporter]: DEBUG: Setting total size of files for upload to: 199
[03:10:49] :	 [Step 6/11] DEBUG [2021-10-04 14:10:49.05]: [Transporter]: INFO: Done computing total size of files to upload.
[03:10:49] :	 [Step 6/11] DEBUG [2021-10-04 14:10:49.06]: [Transporter]: INFO: Starting upload for reservation: (CPUSource/v4/25/63/1a/25631acf-376e-7398-6a8c-e1840347743e/http-transport-diagnostic-2808097841743991954.txt) file: (http-transport-diagnostic-2808097841743991954.txt/1) size: 199
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.03]: [Transporter]: INFO:      File: http-transport-diagnostic-2808097841743991954.txt 199/199, 100.00% completed
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.03]: [Transporter]: INFO: Package upload progress: 100.00% completed
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.26]: [Transporter]: INFO: Finished part upload for: (http-transport-diagnostic-2808097841743991954.txt/1) 884.44 B/s in 0 seconds
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.26]: [Transporter]: INFO: Finished upload for reservation: (CPUSource/v4/25/63/1a/25631acf-376e-7398-6a8c-e1840347743e/http-transport-diagnostic-2808097841743991954.txt) file: (http-transport-diagnostic-2808097841743991954.txt)
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.26]: [Transporter]: DEBUG: SMART-CLIENT: getCurrent(); using pinned data center = contentdelivery01.itunes.apple.com
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.26]: [Transporter]: INFO: Waiting on reservation commit(s) to finish.
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.26]: [Transporter]: DBG-X: Using operation named: commitTestReservation
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.26]: [Transporter]: DBG-X: Apple's web service operation input parameters:
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.26]: [Transporter]: DBG-X:   parameter Application = iTMSTransporter
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.26]: [Transporter]: DBG-X:   parameter BaseVersion = 2.1.0
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.26]: [Transporter]: DBG-X:   parameter iTMSTransporterMode = upload
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.26]: [Transporter]: DBG-X:   parameter NewPackageName = http-transport-diagnostic-20211004031047-775-8259.itmsp
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.26]: [Transporter]: DBG-X:   parameter OSIdentifier = Mac OS X 11.5.1 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-5; jre=14.0.2+12-iTunesOpenJDK-5
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.26]: [Transporter]: DBG-X:   parameter reservations = [CPUSource/v4/25/63/1a/25631acf-376e-7398-6a8c-e1840347743e/http-transport-diagnostic-2808097841743991954.txt]
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.26]: [Transporter]: DBG-X:   parameter TransporterArguments = -m upload -u ********** -p **hidden value** -f /tmp/**********.itmsp -k 100000
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.26]: [Transporter]: DBG-X:   parameter Username = **********
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.26]: [Transporter]: DBG-X:   parameter Version = 2.2.0
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.26]: [Transporter]: INFO: id = 20211004141050-223
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.26]: [Transporter]: DEBUG: SMART-CLIENT: getCurrent(); using pinned data center = contentdelivery01.itunes.apple.com
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.26]: [Transporter]: INFO: iTMSTransporter Correlation Key: ad003644-aab4-4418-b889-6e586cb3ebae-0001
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.26]: [Transporter]: DEBUG: DataCenter: contentdelivery01.itunes.apple.com
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.59]: [Transporter]: DBG-X: Apple's web service operation return value:
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.59]: [Transporter]: DBG-X:   parameter SessionExpiration = 2021-10-08T03:10:50.489Z
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.59]: [Transporter]: DBG-X:   parameter Success = true
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.59]: [Transporter]: INFO: Reservation committed: (CPUSource/v4/25/63/1a/25631acf-376e-7398-6a8c-e1840347743e/http-transport-diagnostic-2808097841743991954.txt) file: (http-transport-diagnostic-2808097841743991954.txt)
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.59]: [Transporter]: INFO: All reservation commit(s) finished.
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.60]: [Transporter]: INFO: 128.47 B/s in 1 second
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DEBUG: Using token file path /Users/**********/Library/Caches/com.apple.amp.itmstransporter/UploadTokens/Mac-Pro.localdomain_**********_**********.itmsp.token
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DEBUG: Attempting exclusive file lock on token file
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DEBUG: Set token: **********/**********
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DBG-X: Memory: [JVM] 913M free, 1024M total, 2048M max [System] (Physical) 100622M free, 163840M total (Swap) 0 free, 0 total
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DBG-X: Deflated 461 bytes to 408
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DBG-X: Using operation named: validateAssets
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DBG-X: Apple's web service operation input parameters:
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DBG-X:   parameter Application = iTMSTransporter
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DBG-X:   parameter BaseVersion = 2.1.0
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DBG-X:   parameter CPUToken = **********/**********
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DBG-X:   parameter Files = [metadata.xml, a2c1b31e42fff2ba55ef7614665972b9.pkg]
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DBG-X:   parameter MetadataChecksum = **********
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DBG-X:   parameter MetadataCompressed = (suppressed)
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DBG-X:   parameter MetadataInfo = {app_platform=osx, primary_bundle_identifier=, device_id=, bundle_identifier=, packageVersion=software5.4, apple_id=321756558, asset_types=[product-archive], bundle_version=, bundle_short_version_string=}
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DBG-X:   parameter OSIdentifier = Mac OS X 11.5.1 (x86_64); jvm=14.0.2+12-iTunesOpenJDK-5; jre=14.0.2+12-iTunesOpenJDK-5
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DBG-X:   parameter PackageName = **********.itmsp
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DBG-X:   parameter PackageSize = 147795333
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DBG-X:   parameter RequestUUID = 72cec2e5-6550-4fd9-b084-7b53d3eb2bef
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DBG-X:   parameter StatisticsClientStartDateTimeZoneISO = 2021-10-04T14:10:50+11:00
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DBG-X:   parameter StreamingInfoList = [file: cc5a879c-4e44-4b4c-b705-f8926ba7407e5915344732380891344.swinfo.plist.gz, content-length: 49548, content-type: application/binary, MD5: 5b711cd1e47601c2c79740afc592ee3e]
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DBG-X:   parameter Throttle = 100000
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DBG-X:   parameter Transport = HTTP
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DBG-X:   parameter TransporterArguments = -m upload -u ********** -p **hidden value** -f /tmp/**********.itmsp -k 100000
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DBG-X:   parameter TxTransportVersions = {SigniantTCP=1.2.0, Aspera=1.2.1, DAV=1.2.0, Signiant=1.2.0, HTTP=1.4.3}
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DBG-X:   parameter Version = 2.2.0
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DBG-X:   parameter iTMSTransporterMode = upload
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: INFO: id = 20211004141050-288
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.61]: [Transporter]: DEBUG: SMART-CLIENT: getCurrent(); using pinned data center = contentdelivery01.itunes.apple.com
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.62]: [Transporter]: INFO: iTMSTransporter Correlation Key: ad003644-aab4-4418-b889-6e586cb3ebae-0001
[03:10:50] :	 [Step 6/11] DEBUG [2021-10-04 14:10:50.62]: [Transporter]: DEBUG: SMART-CLIENT: Host HTTP header: contentdelivery01.itunes.apple.com
[03:10:58] :	 [Step 6/11] DEBUG [2021-10-04 14:10:58.62]: [Transporter]: DBG-X: Apple's web service operation return value:
[03:10:58] :	 [Step 6/11] DEBUG [2021-10-04 14:10:58.62]: [Transporter]: DBG-X:   parameter EnableJWTForAllCalls = false
[03:10:58] :	 [Step 6/11] DEBUG [2021-10-04 14:10:58.62]: [Transporter]: DBG-X:   parameter ShouldUseFeatures = (null)
[03:10:58] :	 [Step 6/11] DEBUG [2021-10-04 14:10:58.62]: [Transporter]: DBG-X:   parameter SessionExpiration = 2021-10-08T03:10:51.405Z
[03:10:58] :	 [Step 6/11] DEBUG [2021-10-04 14:10:58.62]: [Transporter]: DBG-X:   parameter MinimumFileSizeThresholdForCheckum = 104857600
[03:10:58] :	 [Step 6/11] DEBUG [2021-10-04 14:10:58.62]: [Transporter]: DBG-X:   parameter SoftwareMediaDescriptionFormat = binary
[03:10:58] :	 [Step 6/11] DEBUG [2021-10-04 14:10:58.62]: [Transporter]: DBG-X:   parameter SoftwareMediaDescriptionThresholds = {in-app-purchase-content={extensions=[pkg], size=100000000}, ccats={extensions=[pdf], size=100000000}, product-archive={extensions=[pkg], size=100000000}, developer-id-package={extensions=[zip, pkg, dmg], size=100000000}, bundle={extensions=[zip, ipa], size=100000000}, large-icon={extensions=[jpg, png, jpeg], size=100000000}}
[03:10:58] :	 [Step 6/11] DEBUG [2021-10-04 14:10:58.62]: [Transporter]: DBG-X:   parameter ResumeFlag = false
[03:10:58] :	 [Step 6/11] DEBUG [2021-10-04 14:10:58.62]: [Transporter]: DBG-X:   parameter Success = true
[03:10:58] :	 [Step 6/11] DEBUG [2021-10-04 14:10:58.62]: [Transporter]: DBG-X:   parameter StreamingSettings = {LogStreamingEnabled=true, MultipartUploadsEnabled=true, AssetDescriptionStreamingEnabled=false}
[03:10:58] :	 [Step 6/11] DEBUG [2021-10-04 14:10:58.62]: [Transporter]: DBG-X:   parameter ProviderName = **********
[03:10:58] :	 [Step 6/11] DEBUG [2021-10-04 14:10:58.62]: [Transporter]: DBG-X:   parameter FilesToUpload = [metadata.xml, a2c1b31e42fff2ba55ef7614665972b9.pkg]
[03:10:58] :	 [Step 6/11] DEBUG [2021-10-04 14:10:58.62]: [Transporter]: DBG-X:   parameter AssetsToDescribe = [{Role=source, Filename=a2c1b31e42fff2ba55ef7614665972b9.pkg, ShouldDescribeAsset=true, UTI=com.apple.pkg}]
[03:10:58] :	 [Step 6/11] DEBUG [2021-10-04 14:10:58.62]: [Transporter]: DBG-X:   parameter CPUToken = fd3d5c14-b203-4375-b986-ec7ff4da0848/1633317057632
[03:10:58] :	 [Step 6/11] DEBUG [2021-10-04 14:10:58.62]: [Transporter]: DBG-X:   parameter Checksums = {a2c1b31e42fff2ba55ef7614665972b9.pkg=**********}
[03:10:58] :	 [Step 6/11] DEBUG [2021-10-04 14:10:58.62]: [Transporter]: DBG-X:   parameter SigniantLoggingLevel = INFO
[03:10:58] :	 [Step 6/11] DEBUG [2021-10-04 14:10:58.62]: [Transporter]: DBG-X:   parameter FileSizes = {a2c1b31e42fff2ba55ef7614665972b9.pkg=147794872}
[03:10:58] :	 [Step 6/11] DEBUG [2021-10-04 14:10:58.63]: [Transporter]: DBG-X:   parameter NewPackageName = **********-20211003201057-942.itmsp</pre>
</details>

### Environment
<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                                   |
| --------------------------- | ------------------------------------------------------- |
| OS                          | 11.5.1                                                  |
| Ruby                        | 3.0.2                                                   |
| Bundler?                    | false                                                   |
| Git                         | git version 2.21.0                                      |
| Installation Source         | /usr/local/Cellar/fastlane/2.195.0/libexec/bin/fastlane |
| Host                        | macOS 11.5.1 (20G80)                                    |
| Ruby Lib Dir                | /usr/local/Cellar/ruby/3.0.2/lib                        |
| OpenSSL Version             | OpenSSL 1.1.1k  25 Mar 2021                             |
| Is contained                | false                                                   |
| Is homebrew                 | true                                                    |
| Is installed via Fabric.app | false                                                   |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/             |
| Xcode Version               | 12.5.1                                                  |
| Swift Version               | 5.4.2                                                   |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# This file contains the fastlane.tools configuration
# You can find the documentation at https://docs.fastlane.tools
#
# For a list of all available actions, check out
#
#     https://docs.fastlane.tools/actions
#
# For a list of all available plugins, check out
#
#     https://docs.fastlane.tools/plugins/available-plugins
#

# Uncomment the line if you want fastlane to automatically update itself
# update_fastlane

default_platform(:ios)

platform :ios do
  desc "Push a new beta build to TestFlight"
  lane :beta do |options|
    get_certificates
    build_app(workspace: "Unity-iPhone.xcworkspace", scheme: "Unity-iPhone", export_xcargs: "-allowProvisioningUpdates")
    deliver(submit_for_review: options[:submit], force: true, skip_screenshots:true)
    upload_to_testflight(distribute_only: true)
  end
end

```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
app_identifier("************") # The bundle identifier of your app
apple_id("************") # Your Apple email address

itc_team_id("************") # App Store Connect Team ID
team_id("************") # Developer Portal Team ID

# For more information about the Appfile, see:
#     https://docs.fastlane.tools/advanced/#appfile

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.195.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                             | Version      |
| ------------------------------- | ------------ |
| did_you_mean                    | 1.5.0        |
| atomos                          | 0.1.3        |
| rexml                           | 3.2.5        |
| CFPropertyList                  | 3.0.4        |
| claide                          | 1.0.3        |
| colored2                        | 3.1.2        |
| nanaimo                         | 0.3.0        |
| xcodeproj                       | 1.21.0       |
| rouge                           | 2.0.7        |
| xcpretty                        | 0.3.0        |
| terminal-notifier               | 2.0.0        |
| unicode-display_width           | 1.8.0        |
| terminal-table                  | 1.8.0        |
| plist                           | 3.6.0        |
| public_suffix                   | 4.0.6        |
| addressable                     | 2.8.0        |
| multipart-post                  | 2.0.0        |
| word_wrap                       | 1.0.0        |
| optparse                        | 0.1.1        |
| tty-screen                      | 0.8.1        |
| tty-cursor                      | 0.7.1        |
| tty-spinner                     | 0.9.3        |
| artifactory                     | 3.0.15       |
| babosa                          | 1.0.4        |
| colored                         | 1.2          |
| highline                        | 2.0.3        |
| commander                       | 4.6.0        |
| excon                           | 0.85.0       |
| faraday-em_http                 | 1.0.0        |
| faraday-em_synchrony            | 1.0.0        |
| faraday-excon                   | 1.1.0        |
| faraday-httpclient              | 1.0.1        |
| faraday-net_http                | 1.0.1        |
| faraday-net_http_persistent     | 1.2.0        |
| faraday-patron                  | 1.0.0        |
| faraday-rack                    | 1.0.0        |
| ruby2_keywords                  | 0.0.5        |
| faraday                         | 1.8.0        |
| unf_ext                         | 0.0.8        |
| unf                             | 0.1.4        |
| domain_name                     | 0.5.20190701 |
| http-cookie                     | 1.0.4        |
| faraday-cookie_jar              | 0.0.7        |
| faraday_middleware              | 1.1.0        |
| fastimage                       | 2.2.5        |
| gh_inspector                    | 1.1.3        |
| json                            | 2.5.1        |
| mini_magick                     | 4.11.0       |
| naturally                       | 2.2.1        |
| rubyzip                         | 2.3.2        |
| security                        | 0.1.3        |
| xcpretty-travis-formatter       | 1.0.1        |
| dotenv                          | 2.7.6        |
| bundler                         | 2.2.22       |
| simctl                          | 1.6.8        |
| jwt                             | 2.2.3        |
| uber                            | 0.1.0        |
| declarative                     | 0.0.20       |
| trailblazer-option              | 0.1.1        |
| representable                   | 3.1.1        |
| retriable                       | 3.1.2        |
| mini_mime                       | 1.1.1        |
| memoist                         | 0.16.2       |
| multi_json                      | 1.15.0       |
| os                              | 1.1.1        |
| signet                          | 0.16.0       |
| googleauth                      | 0.17.1       |
| httpclient                      | 2.8.3        |
| webrick                         | 1.7.0        |
| google-apis-core                | 0.4.1        |
| google-apis-playcustomapp_v1    | 0.5.0        |
| google-apis-androidpublisher_v3 | 0.11.0       |
| google-cloud-env                | 1.5.0        |
| google-cloud-errors             | 1.2.0        |
| google-cloud-core               | 1.6.0        |
| google-apis-iamcredentials_v1   | 0.7.0        |
| google-apis-storage_v1          | 0.7.0        |
| rake                            | 13.0.6       |
| digest-crc                      | 0.6.4        |
| google-cloud-storage            | 1.34.1       |
| emoji_regex                     | 3.2.2        |
| jmespath                        | 1.4.0        |
| aws-partitions                  | 1.506.0      |
| aws-eventstream                 | 1.2.0        |
| aws-sigv4                       | 1.4.0        |
| aws-sdk-core                    | 3.121.1      |
| aws-sdk-kms                     | 1.48.0       |
| aws-sdk-s3                      | 1.103.0      |
| tsort                           | 0.1.0        |
| uri                             | 0.10.1       |
| set                             | 1.0.1        |
| forwardable                     | 1.3.2        |
| logger                          | 1.4.3        |
| pathname                        | 0.1.0        |
| shellwords                      | 0.1.0        |
| cgi                             | 0.2.0        |
| date                            | 3.1.0        |
| timeout                         | 0.1.1        |
| stringio                        | 3.0.0        |
| openssl                         | 2.2.0        |
| io-nonblock                     | 0.1.0        |
| ipaddr                          | 1.2.2        |
| io-wait                         | 0.1.0        |
| zlib                            | 1.1.0        |
| resolv                          | 0.2.0        |
| securerandom                    | 0.1.0        |
| digest                          | 3.0.0        |
| time                            | 0.1.0        |
| open-uri                        | 0.1.0        |
| mutex_m                         | 0.1.1        |
| net-protocol                    | 0.1.0        |
| ostruct                         | 0.3.1        |
| english                         | 0.7.1        |
| erb                             | 2.2.0        |
| strscan                         | 3.0.0        |
| abbrev                          | 0.1.0        |
| io-console                      | 0.5.7        |
| tempfile                        | 0.1.1        |
| delegate                        | 0.2.0        |
| fileutils                       | 1.5.0        |
| tmpdir                          | 0.1.2        |
| base64                          | 0.1.0        |
| singleton                       | 0.1.1        |
| net-http                        | 0.1.1        |
| open3                           | 0.1.1        |
| nkf                             | 0.1.0        |
| prettyprint                     | 0.1.0        |
| pp                              | 0.1.0        |
| find                            | 0.1.0        |
| yaml                            | 0.1.1        |
| psych                           | 3.3.0        |
</details>

*generated on:* **2021-10-04**
</details>

**Comment by twomedia** [comment]
One thing I have noticed is that the app is marked as 'platform: osx' in the deliver summary although I explicitly define `ios` in the fastfile. Might be fixed once #19417 is addressed?
